PathfinderAPI/PathfinderAPI.csproj

```diff
@@ -48,6 +48,9 @@
       <HintPath>..\libs\HacknetPathfinder.exe</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="HacknetPathfinder">
+      <HintPath>..\..\..\..\..\..\Program Files (x86)\Steam\steamapps\common\Hacknet\HacknetPathfinder.exe</HintPath>
+    </Reference>
```

Comment:
This should not be in the PathfinderAPI project, you should remove this file from the commit and force push.